include/z64subs.h

```diff
@@ -10,16 +10,28 @@ extern Vec3f gOneVec3f;
 
 #define SUBS_TIME_PATHING_ORDER 3
 
+// NOTE: all on screen checks are bugged, and always evaluate to true, even if off screen
+typedef enum {
+    /* 0 */ SUBS_OFFER_MODE_NONE,
+    /* 1 */ SUBS_OFFER_MODE_GET_ITEM,
+    // Furthur modes all deal with offering exhange items
+    /* 2 */ SUBS_OFFER_MODE_NEARBY,
+    /* 3 */ SUBS_OFFER_MODE_ONSCREEN,
+    /* 4 */ SUBS_OFFER_MODE_AUTO,
+    /* 5 */ SUBS_OFFER_MODE_AUTO_TARGETED, // Also checks nearby and on screen
+    /* 6 */ SUBS_OFFER_MODE_AUTO_NEARBY_ONSCREEN,
+    /* 7 */ SUBS_OFFER_MODE_MAX
+} SubSOfferMode;
+
 typedef enum {
     /* 0 */ SUBS_CUTSCENE_WITH_PLAYER,
     /* 1 */ SUBS_CUTSCENE_NORMAL,
     /* 2 */ SUBS_CUTSCENE_WITH_PLAYER_SET_FLAG
 } SubSCutsceneType;
 
-//! TODO: rename based on func_8013E748 and func_800B8500
-typedef s32 (*func_8013E748_VerifyFunc)(struct PlayState*, Actor*, void*);
+typedef s32 (*VerifyExchangeItemActor)(struct PlayState*, Actor*, void*);
```

Comment:
```suggestion
typedef s32 (*VerifyExchangeActor)(struct PlayState*, Actor*, void*);
```
Might be fine?

Reply:
Still think it should involve `TalkExchange` since that is the function it uses.